### PR TITLE
add lavalink client 

### PR DESCRIPTION
**Pull request by Tomato6966**
Hello.
I'd like to apply to add my own lavalink-client for v4 only.

In the past I've been maintaining my fork of erela.js, after some time, I decided to create a similar project with different branding and far more features + optimizations to not confess hate or other things.
I'm like 95% done for lavalink-features. and the only things that might change are: couple internal typing names. Or I'm gonna add features. In the docs it's state that i EST to finish it end of august, but I think i'll be done way earlier (in fact i already use it in production)
Also I'm currently writing excessive docs for it, so no worries.
It's both for beginners and profeciants. For beginners due to it's automated features like: Channel Delete detection, automated queue, simple typing style (almost the same like on lavalink) Validators and verificators (for example for links, plugins, sources). Which leads to less-errors during production. It's also suitable for advanced projects, hence it supports remote queue store(s) for stuff like redis, dragonflydb, psql, etc. etc. and included autoplay functions as well as high memory efficiency and modern code-style. That's why I chose to code it in typescript to support: old cjs, mjs, esm and ts code projects.

Additionally, if I get approved, I'd be greatful to be contacted on discord (chrissy8283) to retrieve my "own support channel" with Github integration webhook. Thank you very much.

If you want me to not name it "lavalink-client", I am flexible and can change it.
*Note the contributer of the github project, didn't do anything of that lavalink client, only for an old project, who then made poru.*

Also I applied sorting algorithmn to the Lavalink Clients:

1st Java Based Languages
then Languages sorted by alphabet
then Client-Names sorted by alphabet

Thanks in advance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,36 +98,37 @@ Version numbers can come in different combinations, depending on the release typ
...
+|------------------------------------------------------------------|----------|--------------------------------------------|----------------------------------------|
+| [Lavalink.kt](https://github.com/DRSchlaubi/Lavalink.kt)         | Kotlin   | Kord/JDA/**Any**                           | Kotlin Coroutines                      |
+| [Coglink](https://github.com/PerformanC/Coglink)                 | C        | Concord                                    |                                        |
+| [DisGoLink](https://github.com/disgoorg/disgolink)               | Go       | **Any**                                    |                                        |
+| [Mafic](https://github.com/ooliver1/mafic)                       | Python   | discord.py **V2**/nextcord/disnake/py-cord |                                        |
+| [Lavacord](https://github.com/lavacord/Lavacord)                 | Node.js  | **Any**                                    |                                        |
+| [Lavalink-client](https://github.com/tomato6966/lavalink-client) | Node.js  | **Any Discord Wrapper** (TS, (c/m)JS, ESM) | v4-only, automated, custom queue-store |
```
Comment (topi314) [inline README.md]: I don't think this additional information belongs there

**Review by topi314 — CHANGES_REQUESTED** [review]
the existing order has been fine

File: README.md (hunk 1 above)
Comment (topi314) [reply README.md]: and an **Any** for compatible with is enough aswell

Comment (Tomato6966) [reply README.md]: Ok

Comment (Tomato6966) [reply README.md]: check it now please

**Review by topi314 — APPROVED** [review]

**Comment by Tomato6966** [comment]
Thank you :3

**Pull request merged**